src/operations/query/r4.js

```diff
@@ -41,6 +41,16 @@ module.exports.buildR4SearchQuery = (resourceName, args) => {
     if (args['onset_date'] && !args['onset-date']) {
         args['onset-date'] = args['onset_date'];
     }
+    if (args['_lastUpdated:below']) {
```

Comment:
@imranq2 this block is the main change

Reply:
Is there a reason we can't just send the proper search parameter from the UI instead of changing it here?  Reason i ask is that _lastUpdated:below is not valid FHIR search parameter AFAIK.

Reply:
@imranq2 it is a FHIR modifier 
https://build.fhir.org/search.html#modifiers

Reply:
I will work on another option to handle all of this in the front end and not worry about it in this file.